Source/Core/Core/ConfigManager.h

```diff
@@ -95,7 +95,7 @@ struct SConfig : NonCopyable
   bool bDSPThread = false;
   bool bDSPHLE = true;
   bool bSyncGPUOnSkipIdleHack = true;
-  bool bForceNTSCJ = false;
+  bool bSelectNTSCAsJapanese = false;
```

Comment:
Ugh, I changed this to NTSCJInVI because I was reading your comment from my email, and edits don't get sent there. I might just wait for some sort of consensus from @sepalani before changing it further.